php/helpers.php

```diff
@@ -44,46 +44,48 @@ function block_field( $name, $echo = true ) {
 	}
 
 	// Cast block value as correct type.
-	switch ( $block_lab_config->fields[ $name ]->type ) {
-		case 'string':
-			$value = strval( $value );
-			break;
-		case 'textarea':
-			$value = strval( $value );
-			if ( isset( $block_lab_config->fields[ $name ]->settings['new_lines'] ) ) {
-				if ( 'autop' === $block_lab_config->fields[ $name ]->settings['new_lines'] ) {
-					$value = wpautop( $value );
+	if ( isset( $block_lab_config->fields[ $name ]->type ) ) {
+		switch ( $block_lab_config->fields[ $name ]->type ) {
```

Comment:
In a previous version ,[1.3.1](https://github.com/getblocklab/block-lab/blob/0dc89aab9b4e05a7d00ecdb063e0f10da5f1400f/php/helpers.php#L46), the switch statement in `block_field()` was wrapped in an `isset()` conditional:

https://github.com/getblocklab/block-lab/blob/0dc89aab9b4e05a7d00ecdb063e0f10da5f1400f/php/helpers.php#L45-L47

This PR adds that `isset()` conditional back in, though maybe there was another reason for removing it.

Reply:
Since `className` isn't in `$block_lab_config['fields']`, this caused an error before.